apps/staking/src/styles/components/table.ts

```diff
@@ -13,7 +13,7 @@ export const Table = {
     variants: {
         simple: {
             th: {
-                backgroundColor: 'gray.900',
+                backgroundColor: 'dark.gray.secondary',
```

Comment:
That `src/styles/*` inside the staking should not be used anymore. We should only reference the one coming from `@explorer/ui`. Do you mind cleaning that up and replacing it where it is used inside staking? That will probably fix a bug where the screen is not automatically starting on dark-mode, but light-mode instead.


cc: @nevendyulgerov (just to be aware)

Reply:
@brunomenezes sure thing, I will cleaning and replacing it up

Reply:
> @brunomenezes sure thing, I will cleaning and replacing it up

For the clean-up, will you do it on this PR? I saw you did fix some of the things I mentioned above.